cmake/MacOSX/README.md

```
@@ -7,3 +7,112 @@ To use TOPP tools as regular applications in your own shell just add the followi
 export OPENMS_TOPP_PATH=<OpenMS-PATH>
 source $OPENMS_TOPP_PATH/.TOPP_bash_profile
 ```
+
+---
+
+## macOS Code Signing and Notarization
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx_closed; Actual: atx

(MD003, heading-style)